ctlearn/default_models/single_cnn.py

```diff
@@ -19,8 +20,10 @@ def single_cnn_model(data, model_params):
                 model.trainable = trainable_backbone
     else:
         sys.path.append(model_params["model_directory"])
-        engine_module = importlib.import_module(model_params["engine"]["module"])
-        engine = getattr(engine_module, model_params["engine"]["function"])
+        engine_module = importlib.import_module(model_params["engine_cnn"]["module"])
+        engine_module_param = importlib.import_module(model_params["engine_prm"]["module"])
```

Comment:
And here:
engine_mlp_module_param = importlib.import_module(model_params["engine_mlp"]["module"])
mlp= multilayer precetron, which is the actual engine used

Reply:
done